http-builder-ng-core/src/test/groovy/groovyx/net/http/tk/HttpPatchTestKit.groovy

```diff
@@ -0,0 +1,625 @@
+/*
+ * Copyright (C) 2017 Keith Suderman
```

Comment:
Realize this is probably a template, but as above this should be "HttpBuilder-NG Project".